tests/__tests__/sitka_utils.test.ts

```diff
@@ -1,19 +1,76 @@
+import { sitka } from "../sitka-test"
+import { hasMethod, getInstanceMethodNames, createAppStore, createStateChangeKey, createHandlerKey } from "../../src/sitka"
```

Comment:
nitpick:
go ahead and break up this line on the commas
`import { hasMethod,
   getInsanceMethodNames,
  createAppStore,
 ...
} from "../../ ....."
`